lib/wrapbox/runner/ecs.rb

```diff
@@ -488,12 +510,16 @@ def build_run_task_options(task_definition_arn, class_name, method_name, args, c
         role_arn = task_role_arn || self.task_role_arn
         overrides[:task_role_arn] = role_arn if role_arn
 
+        additional_placement_constraints = []
+        if ec2_instance_id
+          additional_placement_constraints << { type: "memberOf", expression: "ec2InstanceId == #{ec2_instance_id}" }
+        end
         {
           cluster: cluster || self.cluster,
           task_definition: task_definition_arn,
           overrides: overrides,
           placement_strategy: placement_strategy,
-          placement_constraints: placement_constraints,
+          placement_constraints: (placement_constraints || []) + additional_placement_constraints,
```

Comment:
In my opinion, it is better to assign `[]` on constructor.

Reply:
I changed the code https://github.com/reproio/wrapbox/pull/26/commits/855bd5913bb5111215831e3d9cbaefce6bb70469